presto-parquet/src/test/java/com/facebook/presto/parquet/reader/TestData.java

```diff
@@ -0,0 +1,199 @@
+                    mixedList.add(randomInt(r, bitWidth));
+                }
+            }
+        }
+        int[] result = new int[size];
+        mixedList.getElements(0, result, 0, size);
+        return result;
+    }
+
+    public static Slice randomBigInteger(Random r)
+    {
+        byte[] result = new byte[2 * SIZE_OF_LONG];
+        byte[] high = randomLong(r, 0, 0x4b3b4ca85a86c47aL);
+        System.arraycopy(high, 0, result, 0, high.length);
+        byte[] low = randomLong(r, 0, 0x98a2240000000000L);
+        System.arraycopy(low, 0, result, 2 * SIZE_OF_LONG - high.length, low.length);
+        return Slices.wrappedBuffer(result);
+    }
+
+    private static byte[] randomLong(Random r, long min, long max)
+    {
+        return Longs.toByteArray(r.nextLong() % (max - min) + min);
+    }
+
+    public static int randomInt(Random r, int bitWidth)
+    {
+        checkArgument(bitWidth <= 32 && bitWidth > 0, "bit width must be in range 1 - 32 inclusive");
+        if (bitWidth == 32) {
+            return r.nextInt();
+        }
+        return propagateSignBit(r.nextInt(), 32 - bitWidth);
+    }
+
+    public static int randomUnsignedInt(Random r, int bitWidth)
+    {
+        checkArgument(bitWidth <= 32 && bitWidth >= 0, "bit width must be in range 0 - 32 inclusive");
+        if (bitWidth == 32) {
+            return r.nextInt();
+        }
+        else if (bitWidth == 31) {
+            return r.nextInt() & ((1 << 31) - 1);
+        }
+        return r.nextInt(1 << bitWidth);
+    }
+
+    public static long randomLong(Random r, int bitWidth)
+    {
+        checkArgument(bitWidth <= 64 && bitWidth > 0, "bit width must be in range 1 - 64 inclusive");
+        if (bitWidth == 64) {
+            return r.nextLong();
+        }
+        return BytesUtils.propagateSignBit(r.nextLong(), 64 - bitWidth);
+    }
+
+    public static byte[][] randomBinaryData(int size, int minLength, int maxLength)
+    {
+        Random random = new Random(Objects.hash(size, minLength, maxLength));
+        byte[][] data = new byte[size][];
+        for (int i = 0; i < size; i++) {
+            int length = random.nextInt(maxLength - minLength + 1) + minLength;
+            byte[] value = new byte[length];
+            random.nextBytes(value);
+            data[i] = value;
+        }
+
+        return data;
+    }
+
+    public static byte[][] randomAsciiData(int size, int minLength, int maxLength)
+    {
+        Random random = new Random(Objects.hash(size, minLength, maxLength));
+        byte[][] data = new byte[size][];
+        for (int i = 0; i < size; i++) {
+            int length = random.nextInt(maxLength - minLength + 1) + minLength;
+            byte[] value = new byte[length];
+            for (int j = 0; j < length; j++) {
+                value[j] = (byte) random.nextInt(128);
+            }
+            data[i] = value;
+        }
+
+        return data;
+    }
+
+    private static int propagateSignBit(int value, int bitsToPad)
```

Comment:
why here and in ByteUtils?

Reply:
I looked at the code and found that this can actually be deleted.